autoload/go/debug.vim

```diff
@@ -899,6 +911,49 @@ function! s:isActive()
   return len(s:state['message']) > 0
 endfunction
 
+" List All Goroutines Running
+function! go#debug#Goroutines(...) abort
+  try
+    let l:currentGoroutineId = 0
+    try
+      let l:currentGoroutineId = s:groutineID()
+    catch 
+      echom "current goroutineId not found..."
```

Comment:
Can you change `echom` here to `go#util#EchoWarning`?

Reply:
fixed